modules/ROOT/pages/configure-message-flows.adoc

```diff
@@ -3,24 +3,56 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Message flow diagram here? - Inbound transmission - validate transmission payload - route to message flows - route through message flows
+To receive messages from a partner, create a message flow and configure its sections for both the source and target. For the Endpoint, source at host - this is the origin of the message flow when defined by a host endpoint and message type, often an API. For the endpoint, target at host - this is the final destination of a message flow when defined by a host endpoint and message type, often a Host API.  
```

Comment:
Is this the third configuration topic, after configuring the host and then configuring the partner? If so, use the same language in the title as used in the doc's intro text. Try to make it parallel with the previous two config topic titles.

Reply:
I don't understand the context of this paragraph. What are the "sections" of the message flow? Do we mean the host endpoint, the partner endpoint, and the transformation in between? Is the transformation one or many sections?

Should this paragraph really be an intro to a two-item list, where the items are labels "Endpoint, source at host" and "Endpoint, target at host," and the texts next to them definitions?

Do we need to document this, given that we already introduced the host-side origin, the host-side target, and the partner-side storage?

If these are the same things, try to use the same language as before. Avoid capitalizing anything unless it is a proper noun or you are specifically quoting a UI label...in which case you should use bold.

Reply:
Copyedits:

```suggestion
To receive messages from a partner, create a message flow and configure its endpoints. For *Endpoint, source at host*, specify the origin of the message flow when defined by a host endpoint and message type, often an API. For *Endpoint, target at host*, specify the final destination of a message flow when defined by a host endpoint and message type, often a host API.  
```